docs/faq.md

```diff
@@ -42,6 +42,27 @@ https://docs.docker.com/desktop/
 **A:**
 -->
 
+<!-- #640 -->
+**Q: How can I restart Rancher Desktop?**
+
+**A:** Click **File** and then **Exit**, or right click on the tray icon at the bottom of the screen and choose to quit the app there.
+
+**Q: How can I perform a clean uninstall of Rancher Desktop?**
+
+**A:** You have two options for this procedure:
```

Comment:
actually, performing both the steps would produce a clean(er) uninstall.

Reply:
Added link to installation page instead as advised.